### PR TITLE
.pth file added

### PR DESCRIPTION
**Pull request by pawanw17**
.pth file added but a generic python path is not yet listed in tools/buildTools/CMakelists.txt 
addresses issue #286

### CODE REVIEW

File: tools/buildTools/CMakeLists.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required( VERSION 2.6 )
 
-
+INSTALL(FILES "robocomp.pth" DESTINATION ~/.local/lib/python3.6/site-packages)
```
Comment (orensbruli) [inline tools/buildTools/CMakeLists.txt:3]: Ok, we need to make this to not depend on the python version, but I think I have seen something for this.

Comment (orensbruli) [reply tools/buildTools/CMakeLists.txt:3]: python3 -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])'

File: tools/buildTools/CMakeLists.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required( VERSION 2.6 )
```
Comment (orensbruli) [inline tools/buildTools/CMakeLists.txt:2]:
```suggestion
execute_process(COMMAND python3 -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])' PYTHON_SITE_PATH OUTPUT_STRIP_TRAILING_WHITESPACE)
```

File: tools/buildTools/CMakeLists.txt (hunk 1 above)
Comment (orensbruli) [inline tools/buildTools/CMakeLists.txt:3]:
```suggestion
INSTALL(FILES "robocomp.pth" DESTINATION ${PYTHON_SITE_PATH} )
```

**Comment by orensbruli** [comment]
Could you test the suggestions and make this work that way?

**Review by orensbruli — CHANGES_REQUESTED** [review]

File: tools/buildTools/CMakeLists.txt (hunk 1 above)
Comment (pawanw17) [reply tools/buildTools/CMakeLists.txt:3]: Yeah, I didn't know how to find the path of site-packages... that's why left the PR incomplete

**Review by orensbruli — COMMENTED** [review]
Have you tested the changes? I haven't

**Comment by pawanw17** [comment]
I have updated the PR.

File: tools/buildTools/CMakeLists.txt [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
-
-
+execute_process(COMMAND python3 -c "from distutils.sysconfig import get_python_lib; print (get_python_lib())" OUTPUT_VARIABLE PYTHON_SITE_PATH OUTPUT_STRIP_TRAILING_WHITESPACE)
```
[the remark this answers is not in the document]
Comment (pawanw17) [reply tools/buildTools/CMakeLists.txt:2]: was getting a syntax error when trying to use single quote in the beginning of command, replacing it with a double quote gave another syntax error saying unexpected end of file. basically cmake was getting confused by using nested quotes.

**Pull request merged**